.github/workflows/build.yml

```diff
@@ -18,6 +18,15 @@ jobs:
             FC: gfortran-8
             OCCA_COVERAGE: 1
             OCCA_FORTRAN_ENABLED: 1
+            useCMake: false
+
+          - name: (Ubuntu) CMake + gcc-8
+            os: ubuntu-18.04
+            CC: gcc-8
+            CXX: g++-8
+            OCCA_COVERAGE: 1
+            OCCA_FORTRAN_ENABLED: 1
```

Comment:
The Fortran parts don't include CMake files, neither the library nor the examples/tests. I suggest to disable the Fortran interface in the CMake runs for the time being.